deploy/README.md

```diff
@@ -88,3 +88,55 @@ All commands below should be run from the root directory of the repository.
                  -f deploy/docker-compose-extra.yml \
                  --profile postgres --profile kafka-connect up --build
   ```
+
+# How to add a new package to the Docker images
+
+Let's say you discover that a new dependency (that can be installed
+using `apt-get`) is needed for Feldera.  Here's what you have to do to
+get it included:
+
+1. Edit `build.Dockerfile` and `Dockerfile` in this directory and add
+  the dependency to the list of dependencies; add a comment explaining
+  why it is needed.
```

Comment:
```suggestion
  why it is needed.
  
  Note that the these Dockerfiles serve slightly different use-cases depending on what you're trying to do: `build.Dockerfile` is used to create a uniform CI environment to build the binaries we run (e.g., pipeline manager, fda etc.). `Dockerfile` is used to create the _runtime_ image that users of feldera download and execute (it runs the pipeline manager and compiles pipelines). Most of the dependencies will need to be in both files, given the fact that we also run tests in the environment of `build.Dockerfile`, but in rare cases it can happen that a dependency or system tool would only need to be in one of the files.
```